### ISSUE TITLE
falcosecurity/falco:0.21.0 non-slim version doesn't have /usr/src/falco-latest available to build the module

### ISSUE THREAD

**Issue by jjo** [falcosecurity/falco#1123]
**Describe the bug**

Falco's docker-entrypoint.sh will try to build the module using /usr/src/falco-latest as DKMS sources, which is not present at image.

**How to reproduce it**

Use provided kubernetes manifest against a cluster without prebuilt modules available

**Expected behaviour**

falco-probe module should get properly built

**Environment**

kops deployed Kubernetes 1.15.11 using kops.io stock images

- Falco version:

falcosecurity/falco:0.21.0 docker image

- Installation method:

Deployed using https://github.com/falcosecurity/falco/blob/master/integrations/k8s-using-daemonset/k8s-with-rbac/falco-daemonset-configmap.yaml

**Additional context**

* Container logs:
```
* Setting up /usr/src links from host
* Unloading falco-probe, if present
* Running dkms install for falco
Error! Could not find module source directory.
Directory: /usr/src/falco-latest does not exist.
* Running dkms build failed, couldn't find /var/lib/dkms/falco/latest/build/make.log
* Trying to load a system falco-probe, if present
* Trying to find precompiled falco-probe for 4.9.0-11-amd64
Found kernel config at /host/boot/config-4.9.0-11-amd64
* Trying to download precompiled module from https://s3.amazonaws.com/download.draios.com/stable/sysdig-probe-binaries/falco-probe-latest-x86_64-4.9.0-11-amd64-49e1bbf18d8c76f696aad3d95de94a7e.ko
curl: (22) The requested URL returned error: 404 Not Found
Download failed, consider compiling your own falco-probe and loading it or getting in touch with the Falco community
```
* Indeed there's no `/usr/src/falco-latest` but instead some "hashed" version of it:
```
$ docker run --entrypoint=/bin/bash falcosecurity/falco:0.21.0 -c 'ls -d /usr/src/*'
/usr/src/falco-be1ea2d9482d0e6e2cb14a0fd7e08cbecf517f94
```

**Comment by jjo** [comment]
FYI I worked around above by creating a configmap to override `/docker-entrypoint.sh` with:
```
#!/usr/bin/env bash
if [[ -z "${SKIP_MODULE_LOAD}" ]]; then
  echo "* Setting up /usr/src links from host"

  for i in "$HOST_ROOT/usr/src"/*
  do
      base=$(basename "$i")
      ln -s "$i" "/usr/src/$base"
  done

  ## Hack as 0.21.0 image doesn't have /usr/src/falco-latest but rather /usr/src/falco-<hash>
  test -d /usr/src/falco-latest || ln -sf $(dirname /usr/src/falco-*/dkms.conf) /usr/src/falco-latest
  ##

  /usr/bin/falco-probe-loader
fi

exec "$@"
```

**Comment by leodido** [comment]
Thanks for the workaround @jjo.

Anyway PR #1111 should have solved this issue. Can you please confirm it by testing `falcosecurity/falco:master` docker image?

**Comment by jjo** [comment]
Thanks @leodido, confirming this issue fixed by PR #1111.
/cc @dbarranco @jbianquetti-nami
